vendor/bat-native-ledger/src/bat_client.cc

```diff
@@ -378,7 +378,7 @@ void BatClient::recoverWalletCallback(bool result,
   ledger_->OnRecoverWallet(ledger::Result::LEDGER_OK, properties.balance_, properties.grants_);
 }
 
-void BatClient::getGrant(const std::string& lang, const std::string& forPaymentId) {
+void BatClient::getGrant(const std::string& lang, const std::string& forPaymentId, const std::string& safetynetToken) {
```

Comment:
how are we ending up with all these camelcase args? I realize one of them was already here, but we should be converting them as we come across them, not following the incorrect convention

Reply:
it's just a lint issue. Method names should be camelcase starting with uppercase and args/variables should use underscores. The lib started out with the wrong conventions and we should be trying to convert them over to match chromium conventions